src/main/java/bio/terra/cli/businessobject/User.java

```diff
@@ -54,9 +54,17 @@
   private ServiceAccountCredentials petSACredentials;
 
   // these are the same scopes requested by Terra service swagger pages
-  private static final List<String> SCOPES =
+  private static final List<String> USER_SCOPES =
       Collections.unmodifiableList(Arrays.asList("openid", "email", "profile"));
 
+  // these are the same scopes requested by Terra service swagger pages, plus the cloud platform
+  // scope. pet SAs need the cloud platform scope to talk to GCP directly (e.g. to check the status
+  // of an AI notebook)
+  public static final List<String> PET_SA_SCOPES =
+      Collections.unmodifiableList(
```

Comment:
What's the effective difference between this and `ImmutableList`?

Reply:
Same thing.